lib/contract_interact/EIP20TokenMock.js

```diff
@@ -0,0 +1,126 @@
+//"use strict";
+
+/**
+ *
+ * This is a utility file which would be used for executing all methods on Pricer contract.<br><br>
+ *
+ * @module lib/contract_interact/pricer
+ *
+ */
+const rootPrefix = '../..';
+const helper = require(rootPrefix + '/lib/contract_interact/helper');
+const coreAddresses = require(rootPrefix + '/config/core_addresses');
+const web3RpcProvider = require(rootPrefix + '/lib/web3/providers/rpc');
+const coreConstants = require(rootPrefix + '/config/core_constants');
+const contractName = 'eip20tokenmock';
+const contractAbi = coreAddresses.getAbiForContract(contractName);
+const GAS_LIMIT = coreConstants.OST_GAS_LIMIT;
+const currContract = new web3RpcProvider.eth.Contract(contractAbi);
+
+
+/**
+ * @constructor
+ *
+ */
+const MockToken = module.exports = function (mockTokenAddress) {
+  this.contractAddress = mockTokenAddress;
+};
+
+MockToken.prototype = {
+
+  /**
+  * Get balance of address
+  *
+  * @return {Promise}
+  *
+  */
+  balanceOf: async function (ownerAddress) {
+    const transactionObject = currContract.methods.balanceOf(ownerAddress);
+    const encodedABI = transactionObject.encodeABI();
+    const transactionOutputs = helper.getTransactionOutputs(transactionObject);
+    const response = await helper.call(web3RpcProvider, this.contractAddress, encodedABI, {}, transactionOutputs);
+    return Promise.resolve(response[0]);
+  },
+
+  /**
+  * Set balance
```

Comment:
commenting for input parameters are missing in all methods

Reply:
Added